Lib/test/test_concurrent_futures.py

```diff
@@ -303,6 +303,34 @@ def test_interpreter_shutdown(self):
         self.assertFalse(err)
         self.assertEqual(out.strip(), b"apple")
 
+    def test_submit_after_interpreter_shutdown(self):
+        # Test the atexit hook for shutdown of worker threads and processes
+        rc, out, err = assert_python_ok('-c', """if 1:
+            import atexit
+            @atexit.register
+            def run_last():
+                try:
+                    t.submit(id, None)
+                except RuntimeError:
+                    print("runtime-error")
+                    raise
+            from concurrent.futures import {executor_type}
+            if __name__ == "__main__":
+                context = '{context}'
+                if context == "":
```

Comment:
Nit: you can write `if not context:`.

Reply:
Changed to `if not context:` in 6b78fc27824ea0cc493d9195b1987b7698100e5d